assets/src/dashboard/app/views/templateDetails/karma/templateDetails.karma.js

```diff
@@ -267,4 +267,63 @@ describe('CUJ: Creator can browse templates in grid view', () => {
       expect(nextTemplateTitle.innerText).toEqual(nextTemplate.title);
     });
   });
+
+  describe('Action: See template preview from detail template view', () => {
+    beforeEach(async () => {
+      fixture.setFlags({ enableTemplatePreviews: true });
+
+      await fixture.render();
+    });
+
+    it('should trigger template preview when user clicks a related template', async () => {
+      await getTemplatesState();
```

Comment:
Also a unused getTemplatesState call, shouldn't be needed.

Reply:
So, without this, the tests fail because the templates aren't loaded?